CHANGES.rst

```diff
@@ -11,6 +11,8 @@ dev (master)
 * Add a server_hostname parameter to HTTPSConnection which allows for
   overriding the SNI hostname sent in the handshake. (Pull #1397)
 
+* Drop support for EOL Python 2.6 (Pull #1429)
```

Comment:
Change this to (Pull #1429 and Pull #1430) as @pquentin removed a little more Python 2.6 code as well.

Reply:
Thanks, I've updated these three things.